examples.manifest.yaml

```diff
@@ -20,13 +20,43 @@
     port: 9009
 - name: ilp-auth-tls
   lang: go
-  path: examples/auth-and-tls/main.go
+  path: examples/tcp/auth-and-tls/main.go
   header: |-
-    Go client library [docs](https://pkg.go.dev/github.com/questdb/go-questdb-client/v2)
+    Go client library [docs](https://pkg.go.dev/github.com/questdb/go-questdb-client/v3)
     and [repo](https://github.com/questdb/go-questdb-client).
   auth:
     kid: testUser1
     d: 5UjEMuA0Pj5pjK8a-fa24dyIf-Es5mYny3oE_Wmus48
   addr:
     host: localhost
     port: 9009
+- name: http
+  lang: go
+  path: examples/http/basic/main.go
+  header: |-
+    Go client library [docs](https://pkg.go.dev/github.com/questdb/go-questdb-client/v3)
+    and [repo](https://github.com/questdb/go-questdb-client).
+- name: http-auth
+  lang: go
+  path: examples/http/auth/main.go
+  header: |-
+    Go client library [docs](https://pkg.go.dev/github.com/questdb/go-questdb-client/v3)
+    and [repo](https://github.com/questdb/go-questdb-client).
+  #auth:
```

Comment:
Looks like a leftover.

Reply:
I wasn't sure what to do with those fields, since I don't know the mechanics of how this is translated into the autodocs for questdb.io and the cloud samples. My http auth examples use different fields (user/pass & token), and I'm not sure what modifications I need to make to any other libs for this to work

Reply:
This yaml is used in Cloud UI. Currently, we only have ILP/TCP snippets there, so the `auth` section is specific to ILP/TCP.

@insmac @amunra @jerrinot we should add ILP/HTTP snippets, at least for the clients that support it. This patch adds some examples for the Go client, so we should probably do the same in Java/C/Python/Rust clients and then update Cloud UI. WDYT?